scripts/cpu_temp.sh

```diff
@@ -18,7 +18,7 @@ print_cpu_temp() {
     else
       val="$(sensors)"
     fi
-    echo "$val" | sed -e 's/^Tccd/Core /' | awk -v format="$cpu_temp_format$cpu_temp_unit" '/^Core [0-9]+/ {gsub("[^0-9.]", "", $3); sum+=$3; n+=1} END {printf(format, sum/n)}'
+    echo "$val" | sed -e 's/^Tccd/Core /' | awk -v format="$cpu_temp_format º$cpu_temp_unit" '/^Core [0-9]+/ {gsub("[^0-9.]", "", $3); sum+=$3; n+=1} END {printf(format, sum/n)}'
```

Comment:
Probably better to just do

```
@cpu_temp_format "%2.0f º"
```

Reply:
oh well, that's kind of embarrassing I thought it was only for number formatting I'll close the PR

Reply:
how about setting it as default tho